### PR TITLE
Updated README and added docker-compose target

### PR DESCRIPTION
**Pull request by ed-**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,14 @@ Additional helpful utilities include:
   container, but you may find it convenient.
 
 ## Quickstart
-Just run this to create a microservice named `examplesvc`:
+Install zenkit with `go get`:
```
Comment (smousa) [inline README.md]: you don't need to install zenkit to build the microservice

Comment (smousa) [reply README.md]: You may want to move this from 'quickstart' and into a section called, 'installing this package' or something

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Additional helpful utilities include:
   container, but you may find it convenient.
 
 ## Quickstart
-Just run this to create a microservice named `examplesvc`:
+Then, run this to create a microservice named `examplesvc`in `<GOPATH>/src/github.com/<your-org>`:
```
Comment (smousa) [inline README.md]: revert this statement

**Comment by ed-** [comment]
retest this

**Pull request merged**